frontend/src/pages/CollaborationReportForm/Pages/activitySummary.js

```diff
@@ -0,0 +1,551 @@
+import StateMultiSelect from '../../../components/StateMultiSelect';
+// import './activitySummary.scss';
+
+const ActivitySummary = ({ collaborators }) => {
+  // we store this to cause the end date to re-render when updated by the start date (and only then)
+  const [endDateKey, setEndDateKey] = useState('endDate');
+  const {
+    register,
+    watch,
+    setValue,
+    control,
+    // getValues,
+    // clearErrors,
+  } = useFormContext();
+
+  const regionalOrState = watch('regionalOrState');
+  const showStates = regionalOrState === 'state';
+
+  const startDate = watch('startDate');
+  const endDate = watch('endDate');
+  const { connectionActive } = useContext(NetworkContext);
+  const placeholderText = '- Select -';
+  const drawerTriggerRef = useRef(null);
+
+  const setEndDate = (newEnd) => {
+    setValue('endDate', newEnd);
+
+    // this will trigger the re-render of the
+    // uncontrolled end date input
+    // it's a little clumsy, but it does work
+    setEndDateKey(`endDate-${newEnd}`);
+  };
+
+  const deliveryMethodOptions = [
+    { label: 'Email', value: 'email' },
+    { label: 'Phone', value: 'phone' },
+    { label: 'In person', value: 'inPerson' },
+    { label: 'Virtual', value: 'virtual' },
+  ];
+
+  return (
+    <>
+      <Helmet>
+        <title>Activity Summary</title>
+      </Helmet>
+      <IndicatesRequiredField />
+      <Fieldset className="smart-hub-activity-summary smart-hub--report-legend margin-top-4">
+        <div className="margin-top-2">
+          <FormItem
+            label="Activity name"
+            name="activityName"
+            fieldSetWrapper
+          >
+            <TextInput
+              id="activityName"
+              name="activityName"
+              type="text"
+              placeholder="Input text"
+              inputRef={register({
+                required: 'Enter activity name',
+              })}
+            />
+          </FormItem>
+        </div>
+        <div className="margin-y-2">
+          {!connectionActive ? <ConnectionError /> : null }
+          <FormItem
+            label="Collaborating specialists "
+            name="collabReportCollaborators"
+            required={false}
+            tooltipText="TTA staff that you worked with during the activity."
+          >
+            <MultiSelect
+              name="collabReportCollaborators"
+              control={control}
+              required={false}
+              valueProperty="user.id"
+              labelProperty="user.fullName"
+              simple={false}
+              placeholderText={placeholderText}
+              options={collaborators.map((user) => ({
+                // we want the role construction here to match what later is returned from the
+                // database, so we do this weirdo mapping thing here
+                value: user.id, label: user.name, roles: user.roles.map((r) => r.fullName),
+              }))}
```

Comment:
We map to `{ value, label },` but still use `valueProperty="user.id" ` and `labelProperty="user.fullName"`. Are those props doing anything here, or is this relying on a fallback?

Reply:
valueProperty and labelProperty don't appear to be doing anything so I went ahead and removed them.